### PR TITLE
CPopulation 1

### PR DESCRIPTION
**Pull request by erorcun**

### CODE REVIEW

File: src/peds/Population.cpp [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+
+void
+CPopulation::LoadPedGroups()
+{
+	int fd;
+	char line[1024];
+	int nextPedGroup = 0;
+
+	// 272?!? what?!?
+	char modelName[272];
```
Comment (aap) [inline src/peds/Population.cpp]: Probably 256 + 16? Maybe an unused array?

File: src/peds/Population.cpp [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+WRAPPER void CPopulation::Update(void) { EAXJMP(0x4F39A0); }
+WRAPPER CPed *CPopulation::AddPedInCar(CVehicle *vehicle) { EAXJMP(0x4F5800); }
+WRAPPER bool CPopulation::IsPointInSafeZone(CVector *coors) { EAXJMP(0x4F60C0); }
+
+void
+CPopulation::Initialise()
+{
+	debug("Initialising CPopulation...\n");
+
+	ms_nNumCivMale = 0;
```
Comment (Nick007J) [inline src/peds/Population.cpp]: From what I noticed, compiler reordered sequential assignments in almost every initializing function, by replacing "1-2-3-4-5-6-7-8" with "1-8-2-7-3-6-4-5" etc. In this case such initialization will make more "sense".

File: src/peds/Population.cpp (hunk 1 above)
Comment (erorcun) [reply src/peds/Population.cpp]: ok

File: src/peds/Population.cpp (hunk 2 above)
Comment (erorcun) [reply src/peds/Population.cpp]: ok

**Pull request merged**